src/HDF5.jl

```diff
@@ -1924,6 +1925,19 @@ function get_chunk(dset::Dataset)
     ret
 end
 
+# Set a single filter
+function set_filter(p::Properties, filter_id, flags, cd_values...)
+    h5p_set_filter(p::Properties, filter_id, flags, length(cd_values), Cuint[cd_values...])
```

Comment:
```suggestion
    # can pass the tuple cd_values directly starting from Julia 1.5+. We use an array for compatibility with 1.3 
    h5p_set_filter(p::Properties, filter_id, flags, length(cd_values), Cuint[cd_values...])
```

Reply:
I think  a minor comment would be good to add so that when we drop 1.3 support, we could update this

Reply:
FWIW, passing the tuple directly (without wrapping in a `Ref`) leads to this on 1.6.1:
```
MethodError: no method matching unsafe_convert(::Type{Ptr{UInt32}}, ::Tuple{UInt32})
```